hosting_docs/source/backups_dr/4-dr-setup.rst

```diff
@@ -1,4 +1,79 @@
+
+*
+  Setting up and monitoring remote data backups or replication.
+
+*
+  Creating an active or passive secondary site and the necessary automation to failover and restore to this site rapidly.
+
+Since Commcare mobile works offline, a disaster at primary infrastructure may not cause an immediate disruption to mobile worker operations. But it definitely impacts every web operations (for e.g. actions based on reports, SMS reminders and integrations) and will soon clog mobile worker operations as well. 
+
+In this regard, to ensure continuity of a CommCare deployment following an event of outage, you must have a comprehensive Disaster Recovery Solution. The purpose of this document is not to present any single Disaster Recovery solution, but rather to present the possibilitis and components that commcare-cloud provides to enable a Disaster Recovery. The specific solution may vary depending on business needs, IT capacity and DR budget and we recommend you to design a DR solution that meets all of your requirements upfornt as part of initial CommCare deployment. 
+
+
+Setting up a secondary environment
+----------------------------------
+
+To setup a secondary environment, copy the existing environment :ref:`config directory <configure-env>` to another directory and use it to :ref:`install CommCare HQ <deploy-commcarehq>`.
+
+If you plan to have a secondary environment identical to primary environment the only file you will need to update is the inventory file with IP addresses from secondary environment.
+
+If not, you can create another environment directory with updated configuration based on the primary config directory.
+
+
+Remote Backups
+--------------
+
+The commcare-cloud backup tool provides everything to enable remote backups. It takes daily and weekly backups by default and can be configured to take hourly backups. The tool can send backups to a remote S3 compatible service if configured. The documenation on :ref:`backups` has the relevant details on how to setup remote backups and restore from backups.
+
+While the remote backups are the minimum requirement to safeguard the data in the event of a disaster it is not enough for a rapid recovery of the CommCare services. Even though, commcare-cloud is able to provide hourly backups it will not be enough to achive one hour RPO since it is not possible to setup a CommCare environment rapidly in an hour at a secondary site. To enable rapid recovery you must have a secondary environment on standby, setup database replication and a failproof recovery plan and scripts.
+
+Database Replication
+--------------------
+
+Continous Database replication is necessary to enable minimal RPO. While commcare-cloud does not have tooling available to setup continous replication all the databases used in CommCareHQ support replication through various means. To set this up, you should consult database specific documentation and set this up yourself.
```

Comment:
What is `RPO`?

Reply:
```suggestion
Continous Database replication is necessary to enable minimal RPO. While commcare-cloud does not have tooling available to set up continous replication all the databases used in CommCareHQ support replication through various means. To set this up, you should consult database specific documentation and set this up yourself.
```

Reply:
RPO is Recovery Point Objective. This is not abbreviated here with the assumption that the reader has consulted the background docs on DR that were mentioned above.